Wikipedia/Code/PageHistoryViewController.swift

```diff
@@ -483,10 +483,21 @@ class PageHistoryViewController: ColumnarCollectionViewController {
                     cell.isSelected = false
                 }
                 cell.selectionOrder = SelectionOrder(cachedCellContent.selectionOrderRawValue)
+            } else if isTargetOfScroll {
```

Comment:
Might be good to refactor this into a separate method or closure, so you aren't repeating code below.

Reply:
I refactored this, but then realized it's not actually animating the de-selection. Working on that now.

Reply:
Improved the animation code. It was animating, but the border color and part of the button doesn't animate. It's possible to hack them in as animated, but that seemed heavy-handed for this experiment. Instead, we're now just using the background to highlight (which looks decent, to my untrained eyes), and more properly made the highlighting a function on the view, and the VC now just tells it to animate.

Reply:
@mcleinman I'm not seeing your commit, can you try pushing again?

Reply:
Now pushed up, thanks.